### ISSUE TITLE
bug: Yuka patch has not been working since Yuka v4.24

### ISSUE THREAD

**Issue by smtgpm** [ReVanced/revanced-patches#3289]
### Bug description

So the manager still suggests that any version of Yuka can be patched, but I've been trying every update since Yuka 4.24 (I think it was 4.24 the last working version, not sure) and all of them have Yuka not starting up once the patches have been applied. 
I instanll via mount (root)
On apkmirror also there is no apk anymore only bundle so latests updates were only possible via Mount, but I'm pretty sure when there still were apks released after 4.24 it was crashing all the same
It keeps crashing on start, app launches and crashes instantly

Android 12
Root

### Error logs

```shell
Ran 
logcat | grep AndroidRuntime
On termux on my android but no log shows mentioning AndroidRunTime
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Please check if version `4.24` works.

**Comment by smtgpm** [comment]
OK so good thing you made me check. It seems that the last working version is 4.29, but woerd fact, all the newer releases only provide a bundle, not the apk on apkmirror.

The thing, is I usually update the app via the play store, patch the app I have installed on the manager and install them via root (mount). So I guess this started failing from version 4.30.

I did some tests though :

I have installed version 4.30 bundle with apkmirror installer, patched the installed app via the manager and installed via mount, and then the app crashes at startup (and wasn't before patching it)

And finally what I have done is download and install the v4.29 bundle, installed via apkmirror installer, patched via manager and installed via mount, and that did work.

So in conclusion, last working version is 4..29, doesn't matter if it's installed as apk or bundle. The only weird thing is that since 4.29 there are no apks anymore on apkmirror, only bundles., but that might also just be a coencidence
